BankAccount/BankAccount/Account.cs

```diff
@@ -2,62 +2,78 @@
 
 namespace BankAccount
 {
-    public enum AccountStatusOne
+    public enum AccountStatus
     {
         Open,
         Closed
     }
-    class Account
+   
+    public class Account
     {
 
-        public AccountStatusOne StatusOne { get; set; }
-        public string OwnerOne { get; set; }
-        public decimal BalanceOne { get; set; }
+        public AccountStatus StatusOwner { get; set; }
+        public AccountStatus StatusNonExisting { get; set; }
+        public string Owner { get; set; }
+        public string NonExistingClient { get; set; }
+        public decimal Balance { get; set; }
        
-        public Account (string ownerOne, decimal balanceOne)
+        public Account (string owner, string nonExistent, decimal balance)
```

Comment:
nonexistent...what?
ii dai un nume de client sau ceva, in clipa in care istantiezi clasa.
Non-existent client....cand instantiezi clasa, ii asignezi un nume de client si gata.